doc/user/content/sql/create-source/kafka.md

```diff
@@ -304,6 +304,33 @@ As long as any offset continues increasing, Materialize is consuming data from
 the upstream Kafka broker. For more details on monitoring source ingestion
 progress and debugging related issues, see [Troubleshooting](/ops/troubleshooting/).
 
+### Consumer Groups
+
+When materialize consumes from a Kafka topic for a source, it uses an identifying consumer group.
```

Comment:
@guswynn, the context isn't really clear to me. I think I need a little help before I can help here in a non-superficial way. Will reach out.